tests/docker_ssi/test_docker_ssi.py

```diff
@@ -32,8 +32,7 @@ def setup_install_supported_runtime(self):
     @irrelevant(context.library == "java" and context.installed_language_runtime < "1.8.0_0")
     @irrelevant(context.library == "php" and context.installed_language_runtime < "7.0")
     @irrelevant(context.library == "nodejs" and context.installed_language_runtime < "17.0")
-    @irrelevant(context.library >= "python@4.0.0rc1" and context.installed_language_runtime > "3.9.0")
-    @irrelevant(context.library < "python@4.0.0rc1" and context.installed_language_runtime < "3.8.0")
```

Comment:
I think we still need this?

These tests are meant to also work on older versions of the library as well, so we need to assume that at some point in the future this code might run for `ddtrace` 3.x and 2.x